flyteplugins/go/tasks/pluginmachinery/flytek8s/config/config.go

```diff
@@ -136,6 +139,11 @@ type K8sPluginConfig struct {
 	// one, and the corresponding task marked as failed
 	CreateContainerErrorGracePeriod config2.Duration `json:"create-container-error-grace-period" pflag:"-,Time to wait for transient CreateContainerError errors to be resolved."`
 
+	// Time to wait for transient CreateContainerConfigError errors to be resolved. If the
+	// error persists past this grace period, it will be inferred to be a permanent
+	// one, and the corresponding task marked as failed
+	CreateContainerConfigErrorGracePeriod config2.Duration `json:"create-container-config-error-grace-period" pflag:"-,Time to wait for transient CreateContainerConfigError errors to be resolved."`
```

Comment:
We probably would want to add this is a separate PR just for tracking singular changes.

Reply:
This actually [was a separate PR](https://github.com/flyteorg/flyte/pull/4310), but I did some rebasing and now the diff is a bit confused.